src/pages/executive-board.js

```diff
@@ -0,0 +1,227 @@
+import Wrapper from '../components/wrapper'
+import Banner from '../components/banner'
+import BackgroundImage from 'gatsby-background-image'
+import { graphql, useStaticQuery } from 'gatsby'
+import Helmet from 'react-helmet'
+import { makeStyles, useTheme } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+import classnames from 'classnames'
+import Avatar from '@material-ui/core/Avatar'
+import InstagramIcon from '@material-ui/icons/Instagram'
+import TwitterIcon from '@material-ui/icons/Twitter'
+
+const useStyles = makeStyles(theme => ({
+  munTextProperty: {
+    color: '#D90845',
+    fontWeight: 'bold',
+    fontFamily: "'Rubik' , sans-serif",
+    fontSize: 60,
+    lineHeight: '71px',
+  },
+  root: {
+    marginBottom: theme.spacing(4),
+    paddingLeft: theme.spacing(4),
+    paddingRight: theme.spacing(4),
+    [theme.breakpoints.up('md')]: {
+      paddingLeft: theme.spacing(2),
+      paddingRight: theme.spacing(2),
+    },
+    [theme.breakpoints.down('xs')]: {
+      paddingRight: theme.spacing(0),
+      paddingLeft: theme.spacing(0),
+    },
+  },
+  containerCard: {
+    border: '1px solid grey',
+    borderRadius: '12px',
+    marginBottom: theme.spacing(5),
+    marginTop: theme.spacing(5),
+    marginRight: theme.spacing(2),
+    position: 'relative',
+    maxHeight: '450px',
+    '&:hover > div': {
+      background: 'rgba(217, 8, 69, 0.7)',
+    },
+    '&:hover > p': {
+      visibility: 'visible',
+    },
+    '&:hover > img': {
+      filter: `brightness(0.7) drop-shadow(0 0 6px #000)`,
+    },
+    marginLeft: theme.spacing(2),
+    width: '450px',
+    [theme.breakpoints.down('xs')]: {
+      width: '300px',
+      maxHeight: '300px',
+      marginRight: theme.spacing(0),
+      marginLeft: theme.spacing(0),
+    },
+    boxShadow:
+      '10px 20px 38px rgba(0, 0, 0, 0.3), 5px 15px 12px rgba(0, 0, 0, 0.22)',
+  },
+  blogImage: {
+    width: '100%',
+    height: '450px',
+    transition: 'all 0.3s ease-out',
+    [theme.breakpoints.down('xs')]: {
+      height: '300px',
+    },
+    background: '#000',
+    borderRadius: '12px',
+  },
+  bottomText: {
+    position: 'absolute',
+    bottom: 0,
+    width: '100%',
+    color: '#FFF',
+    height: '100px',
+    transition: 'all 0.3s ease-out',
+    borderRadius: '12px',
+    background: 'rgba(0, 0, 0, 0.7)',
+    padding: theme.spacing(0.5),
+  },
+  socialIconsContainer: {
+    position: 'absolute',
+    top: '42%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    visibility: 'hidden',
+  },
+  socialIcon: {
+    background: 'transparent',
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+  icon: {
+    width: theme.spacing(6),
+    height: theme.spacing(6),
+  },
+}))
+
+function Something(props) {
+  const classes = useStyles(props)
+  const theme = useTheme()
+
+  const committees = ['UNSC', 'UNODC', 'DISEC', 'UNCSW', 'WHO', 'AIPPM']
```

Comment:
why you are not importing this data from data folder ??

Reply:
Will correct this in next pr.